src/status_im2/common/contact_list_item/view.cljs

```diff
@@ -1,49 +1,71 @@
 (ns status-im2.common.contact-list-item.view
-  (:require [utils.re-frame :as rf]
-            [react-native.core :as rn]
-            [quo2.core :as quo]
+  (:require [quo2.foundations.typography :as typography]
```

Comment:
i know it was i tough one, but please rollback changes like 

[quo/text {:weight :semi-bold} display-name]

[quo/user-avatar

[quo/icon

alos in status-im2 we use quo alias not quo2

Reply:
imo it's confusing that we're using quo (for quo2) in status_im2 and quo for quo in status_im.

We should just keep it simple :)

Reply:
Everything is confusing in this PR :/